src/esp/assets/ResourceManager.h

```diff
@@ -426,6 +575,69 @@ class ResourceManager {
                                scene::SceneNode* parent,
                                DrawableGroup* drawables);
 
+ private:
+  /**
+   * @brief Load object templates given string list of object template
+   * locations.
+   *
+   * This will take the list of file names currently specified in
+   * physicsManagerAttributes and load the referenced object templates.
+   * @param tmpltFilenames list of file names of object templates
+   */
+  void loadObjectTemplates(const std::vector<std::string>& tmpltFilenames);
+
+  /**
+   * @brief build templates for all primitive objects
+   */
+  void buildPrimObjectTemplates();
```

Comment:
Are we pre-building primitive templates or generating them on request? E.g. `sphere_template = constructTemplate(UV_PSHERE)`.

Reply:
Well, this particular stub was an old direction that i removed in future commits, so i will remove it now.

As far as sythesizing the primitive templates, i plan on synthesizing one of each primitive template type, and using them to synthesize primitve-based physics objects, at least for the near term as i develop the primitive object functionality.